apps/web-giddh/src/app/theme/ngx-date-range-picker/ngx-daterangepicker.component.ts

```diff
@@ -610,7 +695,13 @@ export class NgxDaterangepickerComponent implements OnInit {
         this.updateView();
     }
 
-    setEndDate(endDate) {
+    public setEndDate(endDate): void {
```

Comment:
nullity check for endDate

Reply:
Did you check the whole code? Do you think it's required?